packages/apollo-server-core/src/requestPipelineAPI.ts

```diff
@@ -50,13 +52,17 @@ export interface GraphQLRequestContext<TContext = Record<string, any>> {
   readonly queryHash?: string;
 
   readonly document?: DocumentNode;
+  readonly originalDocumentString?: string;
 
   // `operationName` is set based on the operation AST, so it is defined
   // even if no `request.operationName` was passed in.
   // It will be set to `null` for an anonymous operation.
   readonly operationName?: string | null;
   readonly operation?: OperationDefinitionNode;
 
+  readonly persistedQueryHit?: boolean;
+  readonly persistedQueryRegister?: boolean;
```

Comment:
I was thinking we may want to put `persistedQueryHit` and `persistedQueryRegister` on a `metrics` property (of type `GraphQLRequestMetrics`), to group these together with information about cache hits.

Reply:
Done